src/js/utilities.js

```diff
@@ -0,0 +1,91 @@
+const appendChild = (html, place) => {
+  const element = document.createRange().createContextualFragment(html);
+  document.querySelector(place).appendChild(element);
+};
+
+const store = (data, name) => {
+  localStorage.setItem(name, JSON.stringify(data));
+};
+
+const getFromStorage = (name) => {
+  const storage = JSON.parse(localStorage.getItem(name));
+  const elements = [];
+  if (storage) storage.forEach((element) => elements.push(element));
+  return elements;
+};
+
+const insertTasksIntoDom = (tasks) => {
+  for (let i = 0; i < tasks.length; i += 1) {
+    appendChild(
```

Comment:
It would be a good idea to use `forEach` instead of the regular `for loop`, to keep things simpler and clearer.

Reply:
It's replaced with forEach.